docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -2625,11 +2713,82 @@ In some cases, you might want to create multiple styled components in a file ins
 
 The API is similar to JSS `makeStyles` but works with emotion.
```

Comment:
Should we note here that `styled-components` does not work with TSS?

Reply:
Yep, done.